src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -66,7 +68,7 @@ private class Emit(
   tAggInOpt: Option[TAggregable],
   nSpecialArguments: Int) {
 
-  val methods: mutable.Map[IRFunction, MethodBuilder] = mutable.Map()
+  val methods: mutable.Map[String, Seq[(Seq[Type], MethodBuilder)]] = mutable.Map().withDefaultValue(Seq())
```

Comment:
@catoverdrive  We can use a MultiMap here too

Reply:
Nice